docs/create-and-manage-a-shop.md

```diff
@@ -0,0 +1,101 @@
+# Creating and managing a shop
+
+## The basics
+Congratulations on a successful installation of Open Commerce. Now, you can access the Admin Portal at http://localhost:4080/
+![Admin](_assets/01-admin-clean.png). The Open Commerce Admin is the user interface used by administrators and shop managers to set up the required shop settings and manage operations of the shop.
+
+## Create a user
+To create a user, click the profile icon in the top right corner of the screen and select **Create Account**.
+
+
+Click **Create Account** to bring up the sign-up window.
+
+You'll need a valid email to use as your ID. Open Commerce does not enforce any password requirements but we recommend you use a strong password. The first user you create is automatically added to the *account-manager*, *system-manager*, and *owner* groups. The shop owner can assign roles to any subsequent users, but they are not added to any groups automatically.
+
+## Create a shop
+
+Once you've created your first user, you're prompted to create a shop. Enter the name of the shop in the text field and click **Create Shop**. The first shop you create is automatically assigned to the primary shop type. Each Open Commerce installation can have only one *primary shop*. Any subsequent shops you may create is assigned a *merchant shop* type.
+
+You can create multiple shops in Open Commerce. This is referred to as *multitenancy*. To create additional shops, click your shop name in the upper left corner of the screen and choose **New Shop** from the drop-down menu. Enter the name of the new shop and click **Create Shop**. This new shop is automatically assigned to the merchant shop type.
+![Create second shop](_assets/02-admin-create-shop-2.png)
+
+You can switch between shops using the admin portal. Click the name of the current shop in the upper left corner and then choose the shop you want to switch to.
+![Switch shop](_assets/02-admin-switch-shop.png)
+
+Once you have created a user and set up a shop, the admin portal will look like this.
+![Admin](_assets/02-admin-user-shop-setup.png)
+
+## Accounts
+
+You'll find account management and settings options by clicking Accounts in the side menu of the Open Commerce Admin portal.
+
+Within Accounts you can access the following options::
+- Staff Members: View and modify the accounts involved with shop management and the different permission groups to which they belong. The first account created is populated under Owner. You can also create additional groups and invite users here.
+- Customers: View customers of the shop..
+- Invitations: View the outstanding sent invitations. Once the user accepts an invitation it is no longer shown here.
+
+### Create Group
+
+To create a new group, click Create Group under Staff Members, and add the following information in the pop-up modal:
```

Comment:
remove comma after 'and'

Reply:
This line is a recommendation from Missy